test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -396,3 +396,40 @@ Describe "New-Item -Force should throw an error for invalid characters in direct
         { New-Item -Path $invalidPath -ItemType Directory -Force -ErrorAction Stop } | Should -Throw -ErrorId 'CreateDirectoryIOError,Microsoft.PowerShell.Commands.NewItemCommand'
     }
 }
+
+Describe "If the target path contains wildcard characters, the operation should succeed in creating a file link." -Tags @("CI", 'RequireAdminOnWindows') {
```

Comment:
All file link tests are in FileSystem.Tests.ps1 line 555. Please move new tests to the file.